data-transformers/transformers/statsTransformer.py

```diff
@@ -21,6 +21,7 @@
 trialID = sys.argv[5]
 experimentID = trialID.split("_")[0]
 containerPropertiesID = sys.argv[7]
+containerID = filePath.split("/")[-1].split("_")[0]
```

Comment:
You should first save the containers information, and get the generated `uuid` back. Then you can use it in the `io_data` table.